src/relay/pass/type_functor.h

```diff
@@ -67,8 +69,7 @@ class TypeFunctor<R(const Type& n, Args...)> {
   virtual R VisitType_(const IncompleteTypeNode* op, Args... args) TYPE_FUNCTOR_DEFAULT;
 
   virtual R VisitTypeDefault_(const Node* op, Args...) {
-    LOG(FATAL) << "Do not have a default for " << op->type_key();
-    return R();
+    throw Error(std::string("Do not have a default for ") + op->type_key());
```

Comment:
this change is not necessary, just LOG(FATAL) is fine as they also throw and give you stack trace and lineno for easy runtime debugging